service/raft/raft_group0.cc

```diff
@@ -513,12 +536,12 @@ future<> raft_group0::setup_group0(db::system_keyspace& sys_ks, const std::unord
         //
         // In a fully upgraded cluster this should finish immediately (if the network works well) - everyone is in `use_post_raft_procedures`.
         // In a cluster that is currently in the middle of `upgrade_to_group0`, this will cause us to wait until the procedure finishes.
-        group0_log.info("setup_group0: ensuring that the cluster has fully upgraded to use Raft...");
-        if (co_await wait_for_peers_to_enter_synchronize_state(group0_server, _ms, _abort_source, _shutdown_gate.hold())) {
+        group0_log.info("setup_group0: waiting for peers to synchronize state...");
```

Comment:
Well, it's a copy-cat from the method name that you chose. Previously the comment was wrong because it was copy-pasted from a different function, so I just made it relevant to the function name. You could say it's an irrelevant change, but I am changing this code anyway here, so I made it correct. Please feel free to improve these logging messages the way you like, i think this is outside the scope of this patch.